MekHQ/src/mekhq/campaign/market/contractMarket/ContractTerms.java

```diff
@@ -0,0 +1,290 @@
+                commandModifier += 2;
+                salvageModifier += 2;
+                supportModifier += -1;
+                transportModifier += -1;
+            }
+            case PLANETARY_ASSAULT -> {
+                commandModifier += -2;
+                supportModifier += 2;
+                transportModifier += 3;
+            }
+            case RECON_RAID -> {
+                commandModifier += -1;
+                salvageModifier += -2;
+                supportModifier += 1;
+                transportModifier += -1;
+            }
+            case RELIEF_DUTY -> {
+                commandModifier += -1;
+                salvageModifier += 1;
+                supportModifier += 1;
+                transportModifier += 1;
+            }
+            case RIOT_DUTY -> {
+                commandModifier += -2;
+                salvageModifier += 1;
+                supportModifier += 2;
+            }
+            case SECURITY_DUTY -> {
+                commandModifier += -3;
+                supportModifier += 2;
+                transportModifier += 1;
+            }
+        }
+    }
+
+    private void addEmployerModifiers(Faction employer, LocalDate date) {
+        employmentMultiplier = 1.0;
+        if (employer.isSuperPower()) {
+            employmentMultiplier += 0.3;
+            supportModifier += 1;
+            transportModifier += 2;
+        } else if (employer.isMajorPower()) {
+            employmentMultiplier += 0.2;
+            salvageModifier += -1;
+            transportModifier += 1;
+        } else if (employer.isMinorPower()) {
+            employmentMultiplier += 0.1;
+            salvageModifier += -2;
+        } else if (employer.isCorporation() || employer.isMercenary()) {
+            employmentMultiplier += 0.1;
+            commandModifier += -1;
+            salvageModifier += 2;
+            supportModifier += 1;
+            transportModifier += 1;
+        } else if (employer.isIndependent() || employer.isPlanetaryGovt()) {
+            salvageModifier += -1;
+            supportModifier += -1;
+        }
+        if (employer.isStingy()) {
+            employmentMultiplier += -0.2;
+            salvageModifier += -1;
+            supportModifier += -1;
+            transportModifier += -1;
+        }
+        if (employer.isGenerous()) {
+            employmentMultiplier += 0.2;
+            salvageModifier += 1;
+            supportModifier += 2;
+            transportModifier += 1;
+        }
+        if (employer.isControlling()) {
+            commandModifier += -2;
+            salvageModifier += -1;
+        }
+        if (employer.isLenient()) {
+            commandModifier += 1;
+            salvageModifier += 1;
+        }
+        if (date.getYear() < 2781 || date.getYear() > 3062) {
+            salvageModifier += -2;
+        }
+    }
+
+    private void addUnitReputationModifiers(int reputationFactor) {
+        if (reputationFactor <= 0) {
```

Comment:
I think this might benefit from the MathUtility.clam + switch combo I mentioned above.